airbyte-cdk/java/airbyte-cdk/core/src/main/resources/version.properties

```diff
@@ -1 +1 @@
-version=0.8.0
+version=0.8.1
```

Comment:
make sure mssql points to the latest - 0.8.2(?)

Reply:
Done (0.10.4)